### PR TITLE
ENG-1722 Fix for Settings gear not showing on workflow details page

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
This PR fixes an issue where the settings gear stopped showing on the workflow details page.

## Related issue number (if any)
ENG-1722

https://linear.app/aqueducthq/issue/ENG-1722/0015-fix-settings-icon-not-appearing-in-workflow-details-page

## Loom Demo
https://www.loom.com/share/4e6ad7567e7c42e28a65661a9cd98655

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/workflows/workflowHeader.tsx [hunk 1]
```diff
@@ -286,7 +286,7 @@ const WorkflowHeader: React.FC<Props> = ({ user, workflowDag, workflowId }) => {
           <Status status={workflow.dagResults[0].status} />
         </Box>
 
-        <Box sx={{ ml: 2 }}>
+        <Box sx={{ mr: 20 }}>
```
Comment (cw75) [inline src/ui/common/src/components/workflows/workflowHeader.tsx:289]: Do we need to worry about this hard-coded "20"?

**Review by cw75 — APPROVED** [review]

File: src/ui/common/src/components/workflows/workflowHeader.tsx (hunk 1 above)
Comment (vsreekanti) [reply src/ui/common/src/components/workflows/workflowHeader.tsx:289]: The 2 was using the built in MUI theme spacing. Presumably there aren't 20 spacing increments. Is this defaulting to 20 pixels, @agiron123?

Comment (agiron123) [reply src/ui/common/src/components/workflows/workflowHeader.tsx:289]: @vsreekanti nah, this is 160 pixels.

I wouldn't worry about it for now, this stuff will change during the ui-redesign merge most likely.

**Comment by vsreekanti** [comment]
@agiron123, I don't think this actually fixes the issue. The margin that used to be there between the edge of the ReactFlow canvas and the Status Bar is no longer there, and that looks a little wonky IMO.

**Comment by vsreekanti** [comment]
Here's what it looked like in v0.0.14: 
image
image

**Comment by agiron123** [comment]
@vsreekanti okay, i'll take another look. 

This and the other 0.0.15 fixes will probably just go into main and be ignored in our ui-redesign branch.

**Pull request merged**